.github/workflows/docker-rebuild.yml

```diff
@@ -0,0 +1,50 @@
+name: "Docker rebuild"
+on:
+  workflow_dispatch:
+jobs:
+  rebuild-docker-images:
+    name: "Docker rebuild"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Checkout most recent tag"
+      run: |
+        git fetch --tags origin
+        git describe --abbrev=0 | xargs git checkout
```

Comment:
Are we sure this always corresponds to the most recent release? I suppose this will always be triggered from the main branch, so it should give the last tag on the main branch. But there could be more recent tags on other branches, for example patch releases like `v0.61.1`.

At some point we had tags like `v0.55-dev` but those are no longer used AFAICT.

Reply:
Only now I noticed that it is possible to select the tag to be used as an input to the `workflow_dispatch`: 
![image](https://github.com/NatLibFi/Annif/assets/34240031/69978732-cfe1-4343-9caf-f1e8070a0a8a)

 I'll check how that could be used for the build step, so this a flaky way to select the tag for latest release can be removed, and the Annif version for which Docker image to be rebuilt could be freely selected.

Unfortunately the  `workflow_dispatch` trigger can be used only when the workflow in question is already in the default branch, so testing this while in PR is not possible.